stan/math/rev/mat/fun/cholesky_decompose.hpp

```diff
@@ -246,10 +244,8 @@ namespace stan {
       check_symmetric("cholesky_decompose", "A", A);
 
       Eigen::Matrix<double, -1, -1> L_A(value_of_rec(A));
-      Eigen::LLT<Eigen::MatrixXd> L_factor
-        = L_A.selfadjointView<Eigen::Lower>().llt();
+      Eigen::LLT<Eigen::Ref<Eigen::MatrixXd> > L_factor(L_A);
```

Comment:
Good question! I'm not sure if I can do a Ref to a triangular view of a matrix. I'll see if I can figure it out.